server/cmd/server.go

```diff
@@ -39,6 +38,8 @@ import (
 	cliflag "k8s.io/component-base/cli/flag"
 	"k8s.io/component-base/cli/globalflag"
 	"k8s.io/component-base/logs"
+
+	apiextensionapiserver "k8s.io/apiextensions-apiserver/pkg/apiserver"
```

Comment:
strange order

Reply:
at some point need to wire linters into this repo :/  still very much basic stuff only